root/instrument/List.js

```diff
@@ -0,0 +1,47 @@
+const React = require('react');
+const Layout = require('../layout');
+const EntityLink = require('../static/scripts/common/components/EntityLink');
+const {l, lp} = require('../static/scripts/common/i18n');
+
+const instrumentDescription = (description) => ({ __html: " &mdash; " + description });
+const Instrument = (props) => {
+  const instrument = props.instrument;
+  return (
+    <li>
+      <EntityLink entity={instrument} />
+      { instrument.description ? <span dangerouslySetInnerHTML={instrumentDescription(instrument.description)} /> : null }
+    </li>
+  );
+};
+
+const InstrumentList = () => (
+  <Layout>
```

Comment:
You need to set `title={l('Instrument List')}` and `fullWidth={true}` props here.